opendevin/runtime/docker/exec_box.py

```diff
@@ -27,7 +27,7 @@
 
 # FIXME: On some containers, the devin user doesn't have enough permission, e.g. to install packages
 # How do we make this more flexible?
-RUN_AS_DEVIN = config.get(ConfigType.RUN_AS_DEVIN).lower() != 'false'
+RUN_AS_DEVIN = config.get(ConfigType.RUN_AS_DEVIN)
```

Comment:
Please take a look at the PR linked before, it's doing these already. Feel free to check it out and tell if it's missing anything!